vext.go

```diff
@@ -6,9 +6,13 @@ import (
 
 	v "github.com/RussellLuo/validating/v3"
 	"github.com/asaskevich/govalidator"
+	"github.com/th0th/is-email-disposable/pkg/isemaildisposable"
+	"github.com/th0th/is-email-disposable/pkg/service/domain"
```

Comment:
I guess we need to add this new package into [go.mod](https://github.com/RussellLuo/vext/blob/main/go.mod) :)

Reply:
When I did run `go mod tidy`, dependencies got bloated with unnecessarily packages from the `is-email-disposable` package. So I splitted it. It should be fine now.